lib/helpers/analytics_helper.dart

```diff
@@ -35,7 +36,7 @@ class AnalyticsHelper {
   // static const String walletAmountKey = "wallet_amount";
   // static const String buttonPressedKey = "button_pressed";
   // static const String buttonNameKey = "button_name";
-  // static const String screenNameKey = "screen_name";
+  static const String screenNameKey = "screen_name";
```

Comment:
Should we use this one? Or they style from above?

Reply:
Yes, true, I thought an event property might help distinguish the 2 similar properties, but ill leave it out for now